client/my-sites/plans-features-main/media-queries.tsx

```diff
@@ -0,0 +1,31 @@
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+
+const sidebarWidth = 272; //in px
+const plansPageSmallBreakpoint = '780px';
+const plansPageSmallWithSidebarBreakpoint = `${ 780 + sidebarWidth }px`;
+
+/**
+ * Use this sparingly for plans page/section layout purposes.
+ * IMPORTANT: The plans grids do not rely on the same media queries and breakpoints for their rendering.
```

Comment:
This might be due to my lack of context, however I'm wondering if we can expand on these comments for those less familiar with the code.

> Use this sparingly for plans page/section layout purposes.

Can we clarify why this should be used 'sparingly'. Is there a performance concern, or is it because we are planning to remove this soon?

> * IMPORTANT: The plans grids do not rely on the same media queries and breakpoints for their rendering.

Is the 'plans grids' here referring to `FeaturesGrid` in `plans-grid-next/src/components/features-grid` or something else? The plural 'plans grid(s)' makes me wonder what other grids I'm not aware of, so it would be helpful to detail them here.

Reply:
> Can we clarify why this should be used 'sparingly'. Is there a performance concern, or is it because we are planning to remove this soon?

The `IMPORTANT` part is the reason. The grids no longer depend on these screen media queries for being responsive. However, one would be inclined to think that conditionally applying styling based on them is fine (which would be inaccurate). For general page concerns, like margins around headers, toggles, etc. that's fine. For anything grid-related though, they ought to work directly on the grid styles themselves (i.e. use the gridSize props).

I will probably follow up with a P2 post of these changes once finalised

> Is the 'plans grids' here referring to FeaturesGrid in plans-grid-next/src/components/features-grid or something else? The plural 'plans grid(s)' makes me wonder what other grids I'm not aware of 🤔 

Well, we are touching the `ComparisonGrid` in this PR, right? So that's the other one.

Reply:
Updated the doc. Is this a little better now? Happy to clarify further and address any other concerns with this. 🤔

Reply:
Yes thanks for updating.